frontend/src/MovieList.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import Movie from './Movie';
+import PropTypes from 'prop-types';
+
+const MovieList = props => {
+    const movieNodes = props.data.map(movie => (
```

Comment:
try giving meaningful names to the props, so everyone can know what it is, for example this data, it can be named movies